### ISSUE TITLE
Getting 403 response code when running /woo check

### ISSUE THREAD

**Issue by FunkyJamma** [WooMinecraft/WooMinecraft#194]
Hi there Ive used this on a previous server last year. I dont know if it still works but I am getting

[06:49:25] [Server thread/WARN]: [WooMinecraft] Server returned HTTP response code: 403 for URL: https://www.funkycraft.co/wp-json/wmc/v1/server/************
[06:49:25] [Server thread/WARN]: java.io.IOException: Server returned HTTP response code: 403 for URL: https://www.funkycraft.co/wp-json/wmc/v1/server/************

asterisks are to hide the key. 

**Log Link:**

### Server Information:
* **Host:**
* **URL:**
* **WordPress URL:**

### Config.yml
* **url:**

**Comment by spannerman79** [comment]
```
dig +short NS funkycraft.co
zelda.ns.cloudflare.com.
merlin.ns.cloudflare.com.
```
When you created this issue you would have seen the following text
```
// If you are using Cloudflare **STOP** and follow their firewall documentation
// https://bit.ly/2RLBCoE
```

**Comment by FunkyJamma** [comment]
I figured it out, It wasn't Cloudflare I was using the bukkit version for spigot.

**Comment by spannerman79** [comment]
_I had some spare time and I thought I would try and replicate this issue because if this is an actual issue when running a bukkit only plugin on Spigot then documentation needs to be written to reflect this._

Steps I did;
Setup Spigot 1.12.2 server with the WooMinecraft bukkit plugin `WooMinecraft-1.1.2-B1.12.1-R0.1.jar` with vault, Luckperms and testing wordpress install.
It wouldn't even load correctly.  let alone being able to perform a wp-json call.

Removed  `WooMinecraft-1.1.2-B1.12.1-R0.1.jar`, added `WooMinecraft-1.1.2-S1.12.1-R0.1.jar` - no issues. `/woo check` performed as expected.

Then i removed the firewall exception form Cloudflare - and low and behold, _hello http error 403's_.

-----

So in short no, this wasn't because of using a bukkit version within Spigot, but because of a Cloudflare firewall issue.

**Comment by JayWood** [comment]
Thanks @spannerman79 - in the future I'm removing Bukkit anyhow, since Spigot is backwards compat w/ Bukkit from what I can tell.
